portal/templates/flask_user/emails/forgot_password_message.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'flask_user/emails/base_message.html' %}
+
+{% block message %}
+
+{% if app_name == "TrueNTH" %}
+
+<p>It sounds like you're having trouble logging in to TrueNTH.</p>
```

Comment:
You may want to use [`USER_APP_NAME`](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/config.py#L85) instead of instances of "TrueNTH" to keep consistent